aichess.py

```diff
@@ -30,8 +30,8 @@ def __init__(self, iterations: int, baseline: ChessPlayer, testplayers: array, v
         self.__testplayers: array[ChessPlayer] = testplayers
 
     def run(self) -> array:
-        for player in self.__testplayers:
-            self.__run_one_set_of_opponents(player1=player, player2=self.__baseline)
+        for (player, opponent) in zip(self.__testplayers, self.__baseline):
```

Comment:
zip is a nice solution. I would still fix `self.__baseline` to reflect that it is now an array. The name should be plural now. Please also check the ordering of `zip` so we have baseline1 Vs all test players, followed by baseline2 Vs all test players

Reply:
Please also change the parameter name in the constructor to be plural

Reply:
I'm not sure zip does what we want tbh:
`for item in zip([1, 2, 3], ['sugar', 'spice', 'everything nice']):
    print(item)

(1, 'sugar')
(2, 'spice')
(3, 'everything nice')`